Makefile

```diff
@@ -54,7 +58,6 @@ minify: dist/rdflib.min.js
 writable:
 	@sed -i -re 's/git:\/\/github.com\//git@github.com:/' .git/config
 
-.PHONY: test
```

Comment:
I did - I just collected them all at the top, because I find it a bit more readable.